src/block/block_ext.c

```diff
@@ -1389,7 +1419,7 @@ __block_extlist_dump(WT_SESSION_IMPL *session, WT_BLOCK *block, WT_EXTLIST *el,
     if (block->verify_layout)
         level = WT_VERBOSE_NOTICE;
     else
-        level = WT_VERBOSE_DEBUG_2;
+        level = WT_VERBOSE_LEVEL_DEFAULT;
```

Comment:
I like the idea of aligning the debug messages, however, I don't think they should all be level 1. We have 5 debug message levels to minimise noise when debugging. Level 1 should be used for messages that are not too frequent. I think having all of these at level 1 is too noisy.